app/client/templates/HomePage.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Layout } from "@client/components/Layout";
+import { templates } from "@reactivated";
+import { Banner } from "@client/components/homePage/banner/Banner";
+import { About } from "@client/components/homePage/About";
+import { ConferenceInfo } from "@client/components/homePage/conferenceInfo/ConferenceInfo";
+import { GoogleMap } from "@client/components/homePage/GoogleMap";
+import { APIProvider } from "@vis.gl/react-google-maps";
+import { Organizers } from "@client/components/homePage/Organizers";
+import { Sponsors } from "@client/components/homePage/Sponsors";
+
+export const Template = (props: templates.HomePage) => {
```

Comment:
[OPTIONAL] I'd rename the component to `HomePageTemplate`.

Reply:
Unfortunately Reactivated [expects that the export will be named `Template`](https://www.reactivated.io/documentation/concepts/#the-react-side). It also expects that the filename (in this case `HomePage.tsx`) will be the same as the template name in python code.

I could rename it, but then in each template we would need to add `export { HomePageTemplate as Template };`.  So I prefer to leave it as is.